internal/logger/logger_test.go

```diff
@@ -16,6 +17,37 @@ import (
 
 var _ upgrader.Logger = &logger.Logger{}
 
+func testPlan0() ccapi.Plan {
+	return ccapi.Plan{
+		GUID:                   "fake-plan-guid",
+		Name:                   "fake-plan-name",
+		MaintenanceInfoVersion: "fake-plan-version",
+		ServiceOfferingGUID:    "fake-service-offering-guid",
+	}
+}
+
```

Comment:
suggestion(blocking): testInstance2 and testInstance5 are not descriptive. I thought there was an array of some kind. And also one calls the other.

Reply:
This issue has been addressed in the new commit history.